posts/2022/xarray-groupby-vs-geocat-climatology.ipynb

```diff
@@ -2452,8 +2458,31 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.9.15"
+  },
+  "metadata": {
+   "author": "Heather Craker",
+   "date": "2022-11-29",
```

Comment:
It's not showing up in RTD yet. I think this info (and `title` goes in the previous `metadata` entry

Reply:
Oh whoops! I didn't even see that there already was a metadata block. I'll change this now. Thanks for catching it